app/views/practices/form/instructions.html.erb

```diff
@@ -10,6 +10,9 @@
         <section class="margin-bottom-3">
           <%= render partial: "shared/messages", locals: {small_text: false} %>
           <h1 class="font-sans-2xl margin-bottom-205 line-height-sans-205">Instructions</h1>
+          <div class="margin-top-2 margin-bottom-2">
+            <%= render partial: "practices/form/practice_last_updated", locals: {small_text: true} %>
+          </div>
```

Comment:
We can use the class margin-y-2 here, since both the top and bottom margin sizes are identical.